java-gradle/README.md

```diff
@@ -10,17 +10,17 @@ This sample may be out of date. If you face any issues, please reach out to mail
 * [Using docker compose](#using-docker-compose)
 * [Rookout Integration explained](#rookout-integration-explained)
 ## Running local application using Agent-less
-1. Compile the project jar and download the Rookout Java Agent:
+1. Clone and compile the project jar and download the Rookout Java Agent:
      ```bash
-    $ make build
-    ```
-2. Run using your Rookout Token:
-    ```bash
-    $ make run token=YOUR_TOKEN
+    $ git clone https://github.com/Rookout/deployment-examples/tree/master/java-gradle
     ```
+2. Export organization token:
+     ```bash
+     export ROOKOUT_TOKEN=Your-Token
+     ```
 3. Run using gradle:
     ```bash
-    $ make gradle-run token=YOUR_TOKEN
+    $ gradle run
```

Comment:
Use command `./gradlew run`
This uses the gradle wrapper (basically gradle bundled in the repo) so if someone does not have gradle installed they can still run the application

Reply:
Done